documentation/src/main/markdown/releasenotes.md

```diff
@@ -10,11 +10,13 @@
 ### Changed features
 
 * Evicted dependencies in Simple Build Tool(SBT) projects will no longer be included in the Bill of Materials(BoM) generated during the scan.
+* Introduced an optional flag <code>--detect.project.inspector.global.arguments</code> for global flags of Project Inspector. Then, <code>--quite</code> flag and other global flags can be set to this new flag if needed for appropriate pass through to Project Inspector.
```

Comment:
A little message massage based on my understanding of the changes:

* Introduced an optional flag to allow a space-separated list of global options to pass to all invocations of Project Inspector. Specify the <code>--detect.project.inspector.global.arguments</code> flag in the command, followed by other global flags if needed for pass through to Project Inspector.
See [project-inspector properties for further details.](properties/configuration/project-inspector.md)